lib/TimeZoneUtil.php

```diff
@@ -263,3 +168,6 @@ public static function getIdentifiersBC()
         return include __DIR__.'/timezonedata/php-bc.php';
     }
 }
+
+TimeZoneUtil::addTimezoneGuesser(new GuessFromLicEntry());
```

Comment:
Could we move this into a constructor?

Reply:
As there *is* no constructor due to the static nature that is a bit of an issue. I'd love to have that in a constructor but that would mean that the whole class and it's signature needs to change. Any ideas how to solve that?

Reply:
Maybe alternatively move it into a procedural-only file and include it via composer-autoload-file?

Reply:
I've just reworked the whole class into using an instance and exposing the relevant functions via some static functions and a singleton-pattern.

I additionally also reworked the timezone finding into own classes to make extending that possible as well.